app/components/EventItem/RegistrationStatusTag.tsx

```diff
@@ -57,8 +61,7 @@ const RegistrationStatusTag = ({
         </Flex>
       );
     }
-
-    return `Påmelding ${isPastTenseNeeded ? 'ble aldri' : 'er ikke'} bestemt`;
+    return 'Du kan ikke melde deg på';
```

Comment:
```suggestion
    return `Du ${isPastTenseNeeded ? 'kunne' : 'kan'} ikke melde deg på`;
```
😎 idk just a suggestion do what you want you do you

Reply:
nah I was about to remove your other text too cause I think it's overkill but here its at least accurate either way